src/main/java/org/isf/patient/gui/PatientInsertExtended.java

```diff
@@ -547,7 +561,10 @@ private JButton getJOkButton() {
 						}
 					}
 				} else {// Update
-
+					if (!consensus.isConsensusFlag()) {
+						MessageDialog.error(null, "angal.patient.consensus.consensus.mandatory.msg");
+						return;
+					}
```

Comment:
copy this check in the `PatientBrowserManger` into `validatePatient()` method.
It can stay in the gui if you like, but it is not mandatory to have it because the save would be refused by the manager